src/main/resources/filing/templates/small-full-accounts.html

```diff
@@ -1738,100 +1736,302 @@ <h2 class="text--center">Balance sheet</h2>
+
+                      <tr class="figures">
+                          <td id="net-current-assets-label" class="strong">Net current assets:</td>
+
+                          <td class="figure noteIndex" id="uk-busNetCurrentAssetsLiabilitiesCY_ENDNoteIndexes"></td>
+
+                          <td id="net-current-assets-curr-val" class="strong total">
+                              {{if isNotEmpty $current_net_current_assets}}
+                              <ix:nonFraction contextRef="CY_END" decimals="0" unitRef="GBP" name="core:NetCurrentAssetsLiabilities" format="ixt2:numdotdecimal">{{$current_net_current_assets}}</ix:nonFraction>
+                              {{end}}
+                          </td>
+
+                          <td id="net-current-assets-prev-val" class="total">
+                              {{if isNotEmpty $previous_net_current_assets}}
+                              <ix:nonFraction contextRef="PY_END" decimals="0" unitRef="GBP" name="core:NetCurrentAssetsLiabilities" format="ixt2:numdotdecimal">{{$previous_net_current_assets}}</ix:nonFraction>
+                              {{end}}
+                          </td>
+                       </tr>
+                   {{end}}
+
+              <!-- BALANCE SHEET - LIABILITIES - TOTAL ASSETS LESS CURRENT LIABILITIES -->
+
+                  {{if isNotEmpty ($current_total_assets := $liabilities.current_total_assets_less_current_liabilities) ($previous_total_assets := $liabilities.previous_total_assets_less_current_liabilities)}}
+
+                       <tr class="figures">
+                            <td id="total-assets-label">Total assets less current liabilities:</td>
+
+                            <td class="figure noteIndex" id="uk-busTotalAssetsLessCurrentLiabilitiesCY_ENDNoteIndexes"></td>
+
+                            <td id="total-assets-less-current-liabilities-curr-val" class="strong figure">
+                              {{if isNotEmpty $current_total_assets}}
+                                  <ix:nonFraction contextRef="CY_END" decimals="0" unitRef="GBP" name="core:TotalAssetsLessCurrentLiabilities" format="ixt2:numdotdecimal">{{$current_total_assets}}</ix:nonFraction>
+                              {{end}}
+                            </td>
+
+                            <td id="total-assets-less-current-liabilities-prev-val" class="figure">
+                              {{if isNotEmpty $previous_total_assets}}
+                                   <ix:nonFraction contextRef="PY_END" decimals="0" unitRef="GBP" name="core:TotalAssetsLessCurrentLiabilities" format="ixt2:numdotdecimal">{{$previous_total_assets}}</ix:nonFraction>
+                              {{end}}
+                            </td>
+                        </tr>
+                  {{end}}
+
+              <!-- BALANCE SHEET - LIABILITIES - CREDITORS AFTER ONE YEAR -->
+
+                     {{if $creditors_after := $liabilities.creditors_amounts_falling_due_after_more_than_one_year}}
```

Comment:
Can we align this `{{if $creditors_after := $liabilities.creditors_amounts_falling_due_after_more_than_one_year}}` block 
with the beginning of the the previous block `{{if isNotEmpty ($current_total_assets := $liabilities.current_total_assets_less_current_liabilities)`?